setup.py

```diff
@@ -25,7 +27,11 @@
     namespace_packages=["oira", "oira.statistics"],
     include_package_data=True,
     python_requires="~= 3.8",
-    install_requires=["metabase-api", "setuptools"],
+    install_requires=[
+        "metabase-api",
+        "setuptools",
+        "SQLAlchemy[postgresql] >=1.2.999999",
```

Comment:
Not really a blocker but I think we should require SQLAlchemy 1.4, am I wrong?

Reply:
I made sure to match https://github.com/euphorie/Euphorie/blob/main/setup.py#L42
But tbh I didn't check why it's specified in this unusual way.

Reply:
It comes from here: https://github.com/euphorie/Euphorie/commit/4f0bfe23bc34450fed22f30a4da386de9ceda089

We may want to change that to something like  `>= 1.4.0`.

Reply:
That would not have triggered my comment.

Reply:
> I made sure to match https://github.com/euphorie/Euphorie/blob/main/setup.py#L42 But tbh I didn't check why it's specified in this unusual way.

It is not that uncommon https://github.com/Kozea/WeasyPrint/blob/a0de4aba75a9b88e0ffb9da49a7b5579a408f8f5/setup.cfg#L56

The version requirement on `Euphorie` is outdated and maybe perfectible, no discussion about that.

What makes me wonder is why that imperfection has to be propagated while the effort of writing better is so small compared to the discussion it bubbles out.
Anyway, I really would like to decrease the amount of conversation like these onse.

I already approve this PR.